tests/aidstream_3rd_party.py

```diff
@@ -0,0 +1,32 @@
+"""A module to test critical access to aidstream hosted datasets."""
+import pytest
+from lxml import etree
+from web_test_base import WebTestBase
+
+
+class TestAidstreamIsLive(WebTestBase):
+    """Setup URLs for tests."""
+
+    requests_to_load = {
```

Comment:
Suggest adding the aidstream homepage.

Potentially also a dataset over http (depending on what happens).